src/metabase/api/public.clj

```diff
@@ -204,7 +204,7 @@
    general public. Throws a 404 if the Dashboard doesn't exist."
   [& conditions]
   {:pre [(even? (count conditions))]}
-  (-> (api/check-404 (apply t2/select-one [Dashboard :name :description :id :parameters], :archived false, conditions))
+  (-> (api/check-404 (apply t2/select-one [Dashboard :name :description :id :parameters :auto_apply_filters], :archived false, conditions))
```

Comment:
Without this change, public dashboards will not know if they need to show the **Apply** button if `auto_apply_filters` is set to `false`. I also [confirmed with the product people](https://metaboat.slack.com/archives/C04DAKFDTCZ/p1681238761991669?thread_ts=1681233424.166269&cid=C04DAKFDTCZ).